src/rpc/misc.cpp

```diff
@@ -342,15 +342,15 @@ static UniValue signmessagewithprivkey(const JSONRPCRequest& request)
 
 static UniValue setmocktime(const JSONRPCRequest& request)
 {
-            RPCHelpMan{"setmocktime",
-                "\nSet the local time to given timestamp (-regtest only)\n",
-                {
-                    {"timestamp", RPCArg::Type::NUM, RPCArg::Optional::NO, UNIX_EPOCH_TIME + "\n"
-            "   Pass 0 to go back to using the system time."},
-                },
-                RPCResults{},
-                RPCExamples{""},
-            }.Check(request);
+    RPCHelpMan{"setmocktime",
+        "\nSet the local time to given timestamp (-regtest only)\n",
+        {
+            {"timestamp", RPCArg::Type::NUM, RPCArg::Optional::NO, UNIX_EPOCH_TIME + "\n"
+                "   Pass 0 to go back to using the system time."},
+        },
+        RPCResults{},
+        RPCExamples{""},
+    }.Check(request);
```

Comment:
I think for otherwise untouched code we can leave the "bad" whitespace. Otherwise it will increase the git history depth of `git blame`. And this code is in flux anyway, so it can be fixed up the next time someone moves it around.

Reply:
reverted